### PR TITLE
chore: upgrade android lottie to 6.1.0

### PR DESCRIPTION
**Pull request by matinzd**
- Fix autoplay bug
- Upgrade android lottie to 6.1.0
- Remove coroutines

### CODE REVIEW

File: packages/core/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.airbnb.android.react.lottie
 
 import android.graphics.ColorFilter
+import android.util.Log
```
Comment (TheRogue76) [inline packages/core/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt:4]: Is this used?

Comment (matinzd) [reply packages/core/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt]: Nope. Done :)

**Review by TheRogue76 — APPROVED** [review]
Looks good to me. Happy with the update

**Comment by matinzd** [comment]
Fixes #1059

**Pull request merged**